openlibrary/plugins/openlibrary/js/readmore.js

```diff
@@ -11,13 +30,7 @@ export function initReadMoreButton() {
         $(`.${up.attr('class')}.read-more`).removeClass('hidden');
         $(`.${up.attr('class')}.read-less`).addClass('hidden');
     });
-    $('.restricted-view').each(function() {
-        if ($(this).outerHeight()<50) {
-            $(`.${$(this).parent().attr('class')}.read-more`).addClass('hidden');
-        } else {
-            $(this).addClass('restricted-height');
-        }
-    });
+    $(window).on('resize', debounce(resetReadMoreButtons, 50));
```

Comment:
One extra deleted here :)

```suggestion
    resetReadMoreButtons();
    $(window).on('resize', debounce(resetReadMoreButtons, 50));
```